src/WebAudio/Player.js

```diff
@@ -128,7 +157,9 @@ export default class Player extends AudioContext {
   play() {
     const offset = this.pausedAt;
 
-    this.createBufferSource();
+    if (offset !== 0) {
```

Comment:
I guess this line deserves a comment to why this is code is here

Reply:
only create a bufferSource if the the offset is not 0, or do you want to know something else here?

Reply:
Let's take a step back. I'm not referring to just this specific line. I think I'm looking for a more high level description on when AudioBufferSourceNode or when AudioContext is used. What's the purpose of them, when should we be using one over the other. Knowing this, I can probably understand the code in line 160...162 without further detailed comments.

Reply:
I see what you mean. I will create an issue (in YT) do do this later. Documentation on the player can be improved, but the Audio*Node/Context stuff is in general, just general Web Audio API stuff, so nothing special tbh.